basescript/log.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, *streams):
         self.streams = streams
 
     def write(self, data):
-        print data
+        print (data)
```

Comment:
@ManishaBayya  @UmangThapliyal  @RamanjaneyuluIdavalapati  @prasannababuAddagiri I don't see the `Stream` class being used anywhere in the code. Can you please remove it entirely?